content/protocol-adapters/index.textile

```diff
@@ -0,0 +1,30 @@
+---
+title: Protocol Adapters
+meta_description: "Clients can use the Ably network using SSE and MQTT protocol adapters. This is especially useful where an Ably Client Library SDK is not available for your language of choice, or where platform resource constraints prohibit use of a Client Library SDK."
+meta_keywords: "SSE, MQTT, IoT, thin client, protocol adapter, lightweight clients"
+section: protocol-adapters
+index: 0
+jump_to:
+  Help with:
+    - MQTT#mqtt
+    - SSE#sse
+---
+
+While the recommended method for connecting to Ably is through the use of the Ably Client Library SDKs, there are certain situation where this is not possible, due to lack of platform support, or platform resource constraints. In this situation, certain clients may be supported though the use of Ably protocol adapters. There are two protocol adapters in this section of the documentation:
+
+1. MQTT
+2. SSE
+
+Each of these will be described in the following sections.
+
+h2(#mqtt). MQTT
+
+MQTT protocol adapter provides an interface between an MQTT client and Ably. MQTT clients can subscribe and publish to Ably channels. Any client that supports MQTT can therefore leverage Ably's realtime network, with some constraints. 
```

Comment:
Worth mentioning 'IoT' as a use case due to battery and bandwidth constraints but we can skip an in-depth explanation.